boskos/resources.json

```diff
@@ -1,5 +1,822 @@
 [
-{"name": "FAKE_PROJECT_1", "type": "project", "state": "free"},
-{"name": "FAKE_PROJECT_2", "type": "project", "state": "free"},
-{"name": "FAKE_PROJECT_3", "type": "project", "state": "free"}
+  {
```

Comment:
Clean before toss into the free pool sounds sane, I can take that.

Reformat config makes me need to write code to parse the input, rather than I can do a raw unmarshal from json to []resources. What if we'll have more per-resource data, for example, I'll soon need to add maybe some tags to classify gce vs gke projects.